packages/story-editor/src/app/story/useStoryReducer/test/duplicateElementByIds.js

```diff
@@ -169,15 +176,81 @@ describe('duplicateElementById', () => {
       selection: ['789', '456'],
     });
 
-    const result = duplicateElementById({ elementId: '456' });
-    const initalElement = initialState.pages[0].elements.find(
+    const result = duplicateElementsById({ elementIds: ['456'] });
+    const initialElement = initialState.pages[0].elements.find(
       (el) => el.id === '456'
     );
     const newElement = result.pages[0].elements.find(
       (el) => !initialState.pages[0].elements.includes(el)
     );
     expect(result.pages[0].elements).toHaveLength(4);
-    expect(initalElement.x).not.toBe(newElement.x);
-    expect(initalElement.y).not.toBe(newElement.y);
+    expect(initialElement.x).not.toBe(newElement.x);
+    expect(initialElement.y).not.toBe(newElement.y);
+  });
+
+  it('duplicates more than one element but not the background', () => {
```

Comment:
Maybe let's add one with incorrect element IDs as well and verify the initial state doesn't change.